docs/src/app/components/pages/components/DatePicker/ExampleInline.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import DatePicker from 'material-ui/lib/date-picker/date-picker';
+
+export default class DatePickerExampleSimple extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      controlledDate: new Date('2015/07/15'),
```

Comment:
Can we start with a null value? I which we can display the `Controlled Date Input` hint text.